extractFeature.py

```
@@ -0,0 +1,44 @@
+#============================
+#PACKAGES
+#============================
+import sys
+import json
+import time
+import traceback
+from helpers import extractAAC,log,readFile
+
+
+#============================
+#MAIN
+#============================
+def main():
+
+    #Extracting feature from Dataset files
+    try:
+        #Logging start of Program
+        feature = sys.argv[1]
+        dataset = sys.argv[2]
+        
+        log(time.strftime("%m/%d/%Y, %H:%M:%S",time.localtime()))
+        log("Running extractFeature.py  << feature >> "+feature+" << dataset >> "+dataset)
+        
+        #Loading Json
+        err,config=readFile("/config.json")
+        if not err:
```

Just let the Exception raise and catch it.